commit faac3e39870a43c3be6b99169216eedc10a7b5e8
Author: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
Date:   Sat Apr 1 15:59:21 2017 +0200

    pinctrl: meson: meson8b: fix the NAND DQS pins
    
    commit 97ba26b8a9343008504d4e3a87d212bc07b05212 upstream.
    
    The nand_groups table uses different names for the NAND DQS pins than
    the GROUP() definition in meson8b_cbus_groups (nand_dqs_0 vs nand_dqs0).
    This prevents using the NAND DQS pins in the devicetree.
    
    Fix this by ensuring that the GROUP() definition and the
    meson8b_cbus_groups use the same name for these pins.
    
    Fixes: 0fefcb6876d0 ("pinctrl: Add support for Meson8b")
    Signed-off-by: Martin Blumenstingl <martin.blumenstingl@googlemail.com>
    Acked-by: Kevin Hilman <khilman@baylibre.com>
    Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/pinctrl/meson/pinctrl-meson8b.c b/drivers/pinctrl/meson/pinctrl-meson8b.c
index 76f077f..f87ef5a 100644
--- a/drivers/pinctrl/meson/pinctrl-meson8b.c
+++ b/drivers/pinctrl/meson/pinctrl-meson8b.c
@@ -739,8 +739,8 @@ static const char * const sdxc_c_groups[] = {
 static const char * const nand_groups[] = {
 	"nand_io", "nand_io_ce0", "nand_io_ce1",
 	"nand_io_rb0", "nand_ale", "nand_cle",
-	"nand_wen_clk", "nand_ren_clk", "nand_dqs0",
-	"nand_dqs1"
+	"nand_wen_clk", "nand_ren_clk", "nand_dqs_0",
+	"nand_dqs_1"
 };
 
 static const char * const nor_groups[] = {
